### PR TITLE
initial commit - read functionality complete; needs styling

### PR DESCRIPTION
**Pull request by NTR-TheMekaVerse**

### CODE REVIEW

File: src/pages/CatShow.test.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+//         expect(received).toEqual(expected) // deep equality
+
+//         Expected: 1
+//         Received: {}
+
+//             17 |     const catShow = shallow(<CatShow />)
+//             18 |     const catShowCard = catShow.find("Card")
+//         > 19 |     expect(catShowCard).toEqual(1)
+//             |                         ^
+//             20 |   })
```
Comment (WalkerTexas-coder) [inline src/pages/CatShow.test.js]: I am hesitant to let this move onto the main branch. I appreciate that you are attempting to accomplish this and haven't achieved it yet. Is there somewhere else we can store this code or notes for this code that is not the public facing version of our app?

**Review by WalkerTexas-coder — COMMENTED** [review]
This looks good! I think there something to be said about cleaning up the whitespace in your JSX but it's not the end of the world to leave it. Is there anywhere else the incomplete test can be stored for future use? Perhaps you want to just flush the test out before moving on?

File: src/pages/CatShow.test.js (hunk 1 above)
Comment (NTR-TheMekaVerse) [reply src/pages/CatShow.test.js]: I'll clean it up and throw it in the read me under the corresponding developer story

Comment (WalkerTexas-coder) [reply src/pages/CatShow.test.js]: dope

**Review by WalkerTexas-coder — APPROVED** [review]

**Pull request merged**